client/src/components/ErrorMessageComponent/ErrorMessage.js

```diff
@@ -80,7 +92,7 @@ export class ErrorMessageComponent extends Component {
   }
 
   render() {
-    const { showModal, errorMessage, stackErrorMessage, showFixAuth, showAuthSpinner, showAuthStatus, showAuthResponse} = this.state;
+    const { showModal, errorMessage, stackErrorMessage, showFixAuth, showAuthSpinner, showAuthStatus, showAuthResponse, appAdtChinaUrl} = this.state;
```

Comment:
Can we add a space between the Url and the } character?

Reply:
Ready